### ISSUE TITLE
Drift velocity parameter

### ISSUE THREAD

**Issue by paolafer** [next-exp/IC#576]
I have a question regarding the drift velocity: at the moment, we are not using it at the level of hit reconstruction (penthesilea/dorothea), which, by the way, I consider misleading, since we produce hits with a property called Z, which is actually time. We change time to z in a following step, using the drift velocity from time evolution files (for data). If we decide to keep this way, I think we shouldn't mention the drift velocity in the config file of dorothea/penthesilea at all, and keep it as an internal parameter set to 1. In a later stage, the city that makes the geometrical corrections could also make the transit from time to Z.
Alternatively, we could calculate Z in dorothea/penthesilea, but we would need to read the drift velocity somewhere.

**Comment by msorel** [comment]
I think we agreed to keep the same table format for hit information ("RECO" group in hdff5 file) in existing penthesilea city and in future esmeralda city. Right?

In this case, both tables should have Z, Q, E columns, but their meaning will be different:

- Z [units: mm] will assume vdrift = 1 mm/us in oenthesilea, proper drift (~0.892 mm/us, small changes run-by-run) in esmeralda

- Q [units: PEs] will assume no lifetime correction  in penthesilea, proper lifetime correction (as computed from energy plane, I'd say) in esmeralda

- E [units: PEs] will assume no lifetime nor other (geometric, timestamp) correction in penthesilea, all (lifetime, geometric, timestamp) corrections in esmeralda

By the way, at some point we added Ql, El, Qc, Ec, Zc columns to penthesilea. Following the above logic, I understand those have been removed or will be removed, right?

Opinions?
